src/applications/static-pages/contact-chatbot-cta/components/App/index.unit.spec.js

```diff
@@ -6,12 +6,6 @@ import { shallow } from 'enzyme';
 import { App } from '.';
 
 describe('Contact Chatbot CTA <App>', () => {
-  it('does not render when feature toggle is falsey', () => {
-    const wrapper = shallow(<App show={false} />);
-    expect(wrapper.type()).to.equal(null);
-    wrapper.unmount();
-  });
-
```

Comment:
This test was failing after removing the feature flag, but I'm sort of unsure if I should have deleted it.